src/workspacer.Bar/BarForm.cs

```diff
@@ -66,11 +65,17 @@ private void OnLoad(object sender, EventArgs e)
         {
             this.Height = _config.BarHeight;
             var titleBarHeight = this.ClientRectangle.Height - this.Height;
-            this.Location = new Point(_monitor.X, _monitor.Y - titleBarHeight);
+           
+            if (_config.BarIsTop)
+                this.Location = new Point(_monitor.X, _monitor.Y - titleBarHeight);
+            else
+
+                this.Location = new Point(_monitor.X, _monitor.Height - _config.BarHeight);
```

Comment:
```suggestion
                this.Location = new Point(_monitor.X, _monitor.Y + _monitor.Height - _config.BarHeight);
```
This resolved the issue on my setup.